### PR TITLE
Add stylish sidebar component and container implementation

### PR DESCRIPTION
**Pull request by athirahlokman**
@omar-al-hendi conflict resolved.

### CODE REVIEW

**Review by sarahatiqah — APPROVED** [review]

**Review by sarahatiqah — APPROVED** [review]

**Comment by omar-al-hendi** [comment]
Please resolve the conflicts rebase @athirahlokman

**Comment by omar-al-hendi** [comment]
@athirahlokman I think one conflict have been missed 

image

**Comment by sarahatiqah** [comment]
@omar-al-hendi Conflicts resolved

**Comment by omar-al-hendi** [comment]
The sidebar has a small bug, could you fix it ? @athirahlokman 

image

File: src/layout/Container.tsx [hunk 1]
```diff
@@ -1,10 +1,57 @@
...
+        <ul className="sidebar-nav">
+          {linkArrays.map((linkData) => (
+            <li
+              key={linkData.link}
+              className={
+                location.pathname === linkData.link ? "active" : undefined
+              }
+            >
+              <Link to={linkData.link}>{linkData.name}</Link>
+            </li>
```
Comment (omar-al-hendi) [inline src/layout/Container.tsx:47]: you could use here also the navLink componet from react router which has a buit-in active classname

**Review by omar-al-hendi — COMMENTED** [review]
I encourge to look at the UIShell component from carbon Desing System, it could save you time and make the app more consistent. 

Feel free to check: https://react.carbondesignsystem.com/?path=/story/components-ui-shell--fixed-side-nav

**Pull request closed without merging**